app/models/item.rb

```diff
@@ -1,3 +1,3 @@
 class Item < ActiveRecord::Base
-    has_many :facilities, through: :facility_items
+    has_one :facilities, :foreign_key => "facility_id"
```

Comment:
use the hash syntax for symbols: `foreign_key: :facility_id`

Reply:
Changed to hash syntax in item.rb and procedure.rb.